extension/ezjscore/classes/ezjscpacker.php

```diff
@@ -501,6 +501,9 @@ static function fixImgPaths( $fileContent, $file )
            // Pop the css file name
            array_pop( $cssPathArray );
            $cssPathCount = count( $cssPathArray );
+           // sort by url length
+           array_multisort(array_map('strlen', $urlMatches), SORT_DESC, $urlMatches);
```

Comment:
CS. Add spaces around parameters. So for example we think it would be better to use the following instead:
```php
array_multisort( array_map( 'strlen', $urlMatches ), SORT_DESC, $urlMatches );
```